Model/Api/CreateOrder.php

```diff
@@ -638,7 +649,10 @@ protected function getQtyFromTransaction($transactionItem)
      */
     protected function getShippingAmountFromTransaction($transaction)
     {
-        return $transaction->order->cart->shipping_amount->amount;
+        if (isset($transaction->order->cart->shipping_amount->amount)) {
+            return $transaction->order->cart->shipping_amount->amount;
```

Comment:
Maybe we can return to the  (int) or (float) value transformation, do you agree?

Reply:
done